helm/agent/values.yaml

```diff
@@ -86,10 +86,9 @@ volumes: []
 #     secretName: cloudsql-proxy-sa
 # This node is being maintained for legacy reasons.
 service:
-  # Use ClusterIP if your ingress controller routes traffic directly to pods.
-  # Otherwise, update the type based on your ingress controller settings.
-  # For example, defaults for native cloud controllers require NodePort.
-  type: ClusterIP
+  # Defaults for native cloud controllers require NodePort. 
+  # Update the service type based on your ingress controller settings.
+  type: NodePort
```

Comment:
I'm not sure if we should set this as default as NodePorts usage are fairly rare in production, did you run into any issues with ClusterIP?